tests/docker-compose-int-test.yml

```diff
@@ -31,23 +30,28 @@ services:
       retries: 12
     volumes:
       - ./tests/coverage:/app/coverage
+    networks:
+      - ehrbase-net
 
   #
   # Change container name for better readability of logs
   #
   ehrdb:
     container_name: "EHRbase-PG"
+    networks:
+      - ehrbase-net
 
   #
   # Integration tests will be ready after ehrbase is healthy.
   # Will run `runRobotTests`
   #
   ehrbase-integration-tests:
-    image: ${EHRBASE_INTEGRATION_TEST_IMAGE:-ehrbase/integration-tests:latest}
+    image: ${EHRBASE_INTEGRATION_TEST_IMAGE:-ehrbase/integration-tests:latest-feature}
```

Comment:
should this be changed back to `:latest`

Reply:
@alexlehn will change back to **:latest** once pipeline is passed after cleaning up the roles and users from Keycloak.
Before merging to develop, will change to latest tag.